tendermint/tests/lite.rs

```diff
@@ -308,7 +297,13 @@ async fn run_bisection_test(case: TestBisection) {
 
             let expected_state = TrustedState::new(untrusted_signed_header, untrusted_next_vals);
             assert_eq!(new_states[new_states.len() - 1], expected_state);
-            assert_eq!(new_states.len() as i32, expected_num_of_bisections);
+            let _expected_num_of_bisections = case.expected_num_of_bisections;
+            // TODO: number of bisections started diverting in JSON tests and Rust impl
+            // assert_eq!(
+            //     new_states.len() as i32,
+            //     expected_num_of_bisections,
+            //     "expected # bisections"
+            // );
```

Comment:
This is new behaviour. Not sure why these assertions are failing now. Probably the go version @Shivani912 used to generate the new JSON tests moved on and evolved while the current light client in Rust didn't. 

I think it is OK to comment this out for now. It might be debatable if the number of bisections needs to be tested/asserted anyways.

Reply:
> This is new behaviour. Not sure why these assertions are failing now (even for the "happy path" ones). Probably the go version @Shivani912 used to generate the new JSON tests moved on and evolved while the current light client in Rust didn't.
> 
That can't be, because I didn't make any changes in the assertions in test cases. I'll take a closer look at this and fix it. But I think we can merge it this way and do the fix in a separate PR.
> I think it is OK to comment this out for now. It might be debatable if the number of bisections needs to be tested/asserted anyways.
>
Yeah, also it's not possible to make this assertion for Go code because we don't have access to this information. So this is implementation specific in a way.

Reply:
> Yeah, also it's not possible to make this assertion for Go code because we don't have access to this information. So this is implementation specific in a way.

Certainly sounds like, we should remove the assertion here then.

Reply:
opened #276 to track this.